docs/faq.yaml

```
@@ -4,76 +4,102 @@
-    (know your customer, KYC). Once they are validated we do not allow the identity
-    to be edited by the end-user. Is there anything in the roadmap to support this
-    scenario?
-  a: 'There is an open issue for this feature: https://github.com/ory/kratos/issues/47.'
+  - feature
+  q: ' Is the ability to freeze certain identity fields after registration anywhere
+    on the roadmap? Also I want read-only or hidden user-specific data. Can I use
+    traits for that? Example: Some of our identity traits fields may be the subject
+    to identity validation ([know your customer](https://en.wikipedia.org/wiki/Know_your_customer)).
+    Once they are validated we do not allow the identity to be edited by the end-user.'
+  a: 'Currently traits are visible to the users and also editable by them. It makes
+    sense to have read-only and/or hidden traits for users, but we can not give an
+    exact timing when this will be available. There is an open issue for this feature:
+    [Issue](https://github.com/ory/kratos/issues/47).'
 - tags:
   - kratos
   - hooks
+  - feature
   q: 'Are there plans to provide hooks on registration/login/logouts/identity updates? '
-  a: 'Yes! Check these issues for more information on the current progress: https://github.com/ory/kratos/issues/271
-    and https://github.com/ory/kratos/issues/30.'
+  a: 'Yes! See these issues for more information on the current progress: [Feature
+    Request: Webhooks](https://github.com/ory/kratos/issues/271) and [Implement RPC
+    After Hook](https://github.com/ory/kratos/issues/30).'
 - tags:
-  - registration
   - kratos
+  - registration
   - traits
   q: How can I verify beforehand if a username is available during registration?
-  a: You can't right now. It would allow account enumeration attacks.
+  a: You can not right now. It would allow account enumeration attacks. See also the
+    [section in the documentation](https://www.ory.sh/kratos/docs/concepts/security/#account-enumeration).
 - tags:
   - kratos
   - database
   - migration
+  - feature
   q: Do have plans to support user migration scenarios similar to auth0 automatic
     migrations? E.g. configure a callback to the legacy system when you cannot find
     the corresponding user, and store the identity on successful legacy system response.
-  a: No plans yet, but migration is an important piece of the puzzle. We will probably
-    focus on importing existing users first though. An alternative to callback and
-    custom code is probably fronting the legacy system with ORY Hydra (OAuth2/OIDC
-    Server) and then using that as an upstream in ORY Kratos.
+  a: No plans yet, but migration is an important piece of the puzzle. We will focus
+    on importing existing users first. An alternative to callback and custom code
+    is fronting the legacy system with Ory Hydra (OAuth2/OIDC Server) and then using
+    that as an upstream in Ory Kratos.
```

Please wrap **all** multi-line statements like so - what you have there is not valid YAML and will break (did you test rendering the FAQ?):

```suggestion
  a: |
    No plans yet, but migration is an important piece of the puzzle. We will focus
    on importing existing users first. An alternative to callback and custom code
    is fronting the legacy system with Ory Hydra (OAuth2/OIDC Server) and then using
    that as an upstream in Ory Kratos.
```

Please replace all `a: '` multi-line formatted with the pipe syntax as well

---

Rendering the FAQ works fine. The YAML is valid too. 

`npm run gen`
`npm run start`
and working.

I just wanted to make a content update in this PR. 
We need to set up a proper generation pipeline for the FAQ too, it currently lives in one of Kims private repositories. I will open an issue tracking this, but its on my list too. 

As far as I could see the sentiment was that, when the FAQ is in the state we want it (and implement for all projects) we can set up a proper pipeline for outside contributors ect. too.

---

True, looks like it's valid YAML. However, when formatted, the `'` are introduced which need to be escaped if we do `it's` (`it\'s`) which is why it's better to pipe this as I proposed.

---

Yea, you are right, that is why I removed all wordings like "doesn't" ect. 

Certainly not the most elegant solution 😅 . 
I'll fix it now by hand but
will review the script that builds the .yaml next week and make a proposal for a proper pipeline that we can use. 

And if it works well we can open it for outside contribs as well.